.jenkinsfile

```diff
@@ -139,6 +137,12 @@ def run_unit_tests(slave_name, compiler, build_type, model, with_cereal)
                                      description: 'Running ctest failed.')
         throw ex
     }
+    finally
```

Comment:
ah this means whatever happens, exception or not, clean the workspace?

Reply:
yes